android/src/main/java/com/christopherdro/RNPrint/RNPrintModule.java

```diff
@@ -66,6 +66,7 @@ public void print(final ReadableMap options, final Promise promise) {
         final String filePath = options.hasKey("filePath") ? options.getString("filePath") : null;
         final boolean isLandscape = options.hasKey("isLandscape") ? options.getBoolean("isLandscape") : false;
         final String jobName = options.hasKey("jobName") ? options.getString("jobName") : defaultJobName;
+        final String baseUrl = options.hasKey("baseUrl") ? options.getString("baseUrl") : "";
```

Comment:
Can we update the docs with this new property? Should it default to null instead of an empty string it's not passed in ?

Reply:
Resolved
1. Added to README 888918feea94678c26fdc6ac165f8a1609518f2d
2. Changed default to null ab0d9884d319f522e34c879579d9d555a105a13c